engine/orchestration/src/main/java/org/apache/cloudstack/engine/cloud/entity/api/VMEntityManagerImpl.java

```diff
@@ -253,6 +253,8 @@ public void deployVirtualMachine(String reservationId, VMEntityVO vmEntityVO, St
                     }
 
                     _itMgr.start(vm.getUuid(), params, plan, null);
+                }else{
+                    throw ex;
```

Comment:
@GabrielBrascher Maybe I'am wrong but I think is a little bit to much info, the vm is  not the problem in that case. Also throws the _itMgr.start method a couple exception, so that we can not say here when the vm the problem is.

Reply:
@GabrielBrascher, I tend to agree with @DK101010 but would it make sense to cature the exception in a CloudStackException saying that we are "not retrying as no host is given" ? and would that satisfy your requirements?

Reply:
@DaanHoogland @DK101010, I agree that my suggestion might be a bit too much info for this context.
As long as we don't throw raw exceptions at the user's face I am good with it :slightly_smiling_face:.